src/services/message.service.ts

```diff
@@ -1,15 +1,31 @@
 import { BaseChatroomDto, BaseMessageDto, CreateMessageDto } from '@/dtos';
-import { Chatroom, Messages } from '@/entities/nosql';
+import { ChatroomSchema, MessageSchema } from '@/entities/nosql';
 import { HttpException } from '@/exceptions';
 import { model, ObjectId } from 'mongoose';
 
 class MessageService {
   public async getChatroomMessages(id: ObjectId): Promise<BaseMessageDto[]> {
-    const chatroom = model('Chatroom', Chatroom);
+    const chatroom = model('Chatroom', ChatroomSchema);
     const findChatroom: BaseChatroomDto = await chatroom.findOne({ id });
     if (!findChatroom) throw new HttpException(404, 'Chatroom does not exist');
     return findChatroom.messages;
   }
+
+  public async createChatroomMessage(id: ObjectId, userId: number, messageData: CreateMessageDto): Promise<void> {
```

Comment:
nit: I usually like to create the parameter as an object with the given fields.
`createChatroomMessage({id, userId, messageData}: {id:ObjectId, userId: number, messageData: CreateMessageDto})`
This helps avoid you passing in the wrong argument at a given spot